src/urllib3/util/util.py

```diff
@@ -7,9 +7,9 @@ def to_bytes(
 ) -> bytes:
     if isinstance(x, bytes):
         return x
-    elif not isinstance(x, str):
+    if not isinstance(x, str):
```

Comment:
I would argue the `elif not isinstance(x, str)` here is easier to read since this and the above block are semantically linked. Agree that the bottom `if` statement shouldn't be an `elif` though!